.github/workflows/publish.yaml

```diff
@@ -0,0 +1,26 @@
+name: publish
+on:
+  push:
+    branches:
+      - main
+  schedule:
+    - cron:  '0 0 * * *'
```

Comment:
The page is published every day at midnight. This way changes in the submodules are automatically published without manually triggering the pipline in the documentation.